README.md

```diff
@@ -77,7 +77,18 @@ https://www.googleapis.com/auth/admin.directory.user,
 ```
 
 Now that you have a credential that is allowed to the Admin SDK, you can use the
-GSuite provider.
+G Suite provider.
+
+## Configuration
+
+Your G Suite Customer ID is required for some Admins SDK Directory APIs,
+therefore add also `customer_id` when initializing the provider.
```

Comment:
Can you add some info on how users can find this `customer_id`?

Reply:
Removed because instead of the actual customer ID it is possible to use `my_customer`, which would be the G Suite customer ID associated with the impersonating account. This is also specified in https://developers.google.com/admin-sdk/directory/v1/guides/manage-customers#update_an_existing_customer.